vpn/src/main/java/com/duckduckgo/mobile/android/vpn/blocklist/AppTrackerExclusionListUpdateWorker.kt

```diff
@@ -120,10 +119,9 @@ class AppTrackerExclusionListUpdateWorker(
 @ContributesMultibinding(AppScope::class)
```

Comment:
We now need to explicitly tell the bound type, else it won't be picked up
```suggestion
@ContributesMultibinding(
    scope = AppScope::class,
    boundType = LifecycleObserver::class
)
```